subprojects/settings.gradle.kts

```diff
@@ -1,6 +1,6 @@
 @file:Suppress("UnstableApiUsage")
 
-rootProject.name = "avito-android"
+rootProject.name = "avito-android-subprojects"
```

Comment:
Shall this influent on artifacts publication?

Reply:
i'll revert this change; should be done separately